File: ympa_fsharp_functions_server/src/model/UpdateOrderStatusesDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.UpdateOrderStatusDTO

module UpdateOrderStatusesDTO =

  //#region UpdateOrderStatusesDTO

  [<CLIMutable>]
  type UpdateOrderStatusesDTO = {
    [<JsonProperty(PropertyName = "orders")>]
    Orders : UpdateOrderStatusDTO[];
  }

  //#endregion

File: src/gatling/scala/org/openapitools/client/model/UpdateOrderStatusesDTO.scala

package org.openapitools.client.model


case class UpdateOrderStatusesDTO (
    /* Список с обновленными заказами. */
    _orders: List[UpdateOrderStatusDTO]
)
object UpdateOrderStatusesDTO {
    def toStringBody(var_orders: Object) =
        s"""
        | {
        | "orders":$var_orders
        | }
        """.stripMargin
}
